frontend/styles/custom.css

```diff
@@ -33,19 +33,31 @@ header input {
 .bg-cite {
   background: url("/images/pexels-pixabay-159751_scaled.jpg")
 }
+
 .hgf-text-highlight {
   background-color: #cdeefb;
 }
-.hgf-scrollbar {
+
+#hgf-scroll-container {
+  display: flex;
+  height: 200px;
+}
+
+.simplebar-track.simplebar-horizontal {
+  height: 7px !important;
   border-bottom: 1px solid #002864;
 }
-.hgf-scrollbar::-webkit-scrollbar {
-  height: 5px;
+
+.simplebar-horizontal .simplebar-scrollbar {
+  height: 5px !important;
+}
+
+.simplebar-horizontal .simplebar-scrollbar::before {
+  background-color: #05e5ba !important;
+  border-radius: 0px !important;
```

Comment:
> The scrollbar from the Helmholtz homepage has border-radius: 10px which I also used for our scrollbar now. So it has not to be right angle.

The official Helmholtz brand style guide proposes a right angle scroll bar, so I changed it back.